src/firebase/firebase.ts

```diff
@@ -336,6 +336,10 @@ export async function fetchInviteRewardsSenders() {
   return fetchListFromFirebase('inviteRewardAddresses')
 }
 
+export async function fetchCoinbasePaySenders() {
```

Comment:
This seems like a good usecase for using our [address-metadata](https://github.com/valora-inc/address-metadata) to maintain consistent state in Firebase; it looks like the addresses were added manually. Not related to this PR, but it could be good to add a new node to the `address-metadata` for Coinbase Pay addresses and keep the source of truth checked into GH there.